### ISSUE TITLE
Tests fail (with Mojolicious 6.58?)

### ISSUE THREAD

**Issue by eserte** [clustericious/Test-Clustericious-Cluster#8]
My smoker machines started to generate test failures like this one:
http://www.cpantesters.org/cpan/report/0f9fc290-ffa8-11e5-8b6a-b065fcd2507e
I am still gathering more information for better statistical analysis — currently it seems that the failure is caused by Mojolicious 6.58:
```
****************************************************************
Regression 'mod:Mojolicious'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    57913764937141960.00
[1='eq_6.52']         0.0000          0.0000       3.79
[2='eq_6.53']         0.0000          0.0000       7.75
[3='eq_6.54']         0.0000          0.0000      10.50
[4='eq_6.55']         0.0000          0.0000      10.05
[5='eq_6.56']         0.0000          0.0000       4.55
[6='eq_6.57']         0.0000          0.0000       8.74
[7='eq_6.58']        -1.0000          0.0000    -45273272628646696.00

R^2= 1.000, N= 57, K= 8
****************************************************************
```

**Comment by plicease** [comment]
I can reproduce this with Perl 5.23.8 + Mojo 6.58, and confirm that 5.23.8 + Mojo 6.57 is not borked thus.

**Comment by plicease** [comment]
This should be resolved in TCC 0.31 and Clustericious 1.20.
